### PR TITLE
Adds fuzzy cuffs.

### PR DESCRIPTION
**Pull request by DominikPanic**
Yes.

dreamseeker_2019-04-11_20-35-49

:cl: LaKiller8 & ArachnidNexus
imageadd: Adds fuzzy cuffs. Yes, that kind.
/:cl:

### CODE REVIEW

File: code/game/objects/items/handcuffs.dm [hunk 1]
```diff
@@ -144,7 +146,11 @@
 				C.handcuff_update()
 
 
-
+/obj/item/handcuffs/fuzzy
+	name = "fuzzy cuffs"
+	desc = "Use this to keep your partner in line."
```
Comment (Kmc2000) [inline code/game/objects/items/handcuffs.dm]: god why

**Comment by Hughgent** [comment]
Should never have done hand holding. look at where we are now.

**Comment by FloranOtten** [comment]
This adds support for capturing xenomorphs

**Comment by Rohesie** [comment]
Every day we stray further away from god.

**Comment by factoryman942** [comment]
@kevinz000

**Comment by ondrej008** [comment]
> 
> 
> This adds support for capturing xenomorphs
False. Either the regular cuffs could do that, or neither can.

**Comment by spookydonut** [comment]
Appears to be invalid.

**Comment by DominikPanic** [comment]
arachnid died for this

**Comment by AffectedArc07** [comment]
*sigh

**Pull request closed without merging**